config/settings.yml

```diff
@@ -1317,6 +1317,9 @@ rrd:
   event_tracking:
     recipients:
       - fake_email
+  mas_tracking:
+    recipients:
+      - fake_email
```

Comment:
Where does the real email setting go?

Reply:
@yoomlam prod?

Reply:
@yoomlam i assume it goes in the environment of prod. seems to follow the same pattern of other slack notifications

Reply:
"# Add a new feature toggle here to ensure that it is initialized in all environments." in config/features.yml.

I expect this to be available in the Flipper configuration pages when deployed

Reply:
I was referring to the email address that will send it to Slack.